pkg/endpoint/bpf.go

```diff
@@ -489,11 +489,19 @@ func (e *Endpoint) realizeBPFState(regenContext *regenerationContext) (compilati
 		// Compile and install BPF programs for this endpoint
 		if datapathRegenCtxt.regenerationLevel == RegenerateWithDatapathRebuild {
 			err = loader.CompileOrLoad(datapathRegenCtxt.completionCtx, datapathRegenCtxt.epInfoCache, &stats.datapathRealization)
-			e.getLogger().WithError(err).Info("Rewrote endpoint BPF program")
+			if err == nil {
+				e.getLogger().Debug("Rewrote endpoint BPF program")
```

Comment:
This should not happen very often (requires configuration changes). Do we really want to hide any evidence of this from logs?

Reply:
(To clarify, nobody will run debug logging. So if someone hits an issue due to the datapath and we get a sysdump, we won't be able to tell whether any activity is affecting that area)

Reply:
OK, I'll change it back to Info level.